flow/flow.go

```diff
@@ -693,6 +743,49 @@ func PacketsFromGoPacket(packet *gopacket.Packet, outerLength int64, t int64, bp
 		return flowPackets
 	}
 
+	var ipFragStat IPFragmentStat
+	if IPv4Defragmentation {
+		ip4Layer := (*packet).Layer(layers.LayerTypeIPv4)
```

Comment:
I'm not sure we should have this here. I would prefer to put this in network layer